Dockerfile

```diff
@@ -11,8 +11,24 @@ RUN apt-get update && apt-get install -y build-essential \
     python3-pip \
     wireguard \
     iproute2 \
-    openresolv \
-    nmap
+    openresolv
+
+WORKDIR /tmp
+
+RUN apt-get install wget
+RUN wget https://nmap.org/dist/nmap-7.95.tar.bz2
+
+RUN tar jxvf nmap-7.95.tar.bz2
+
+WORKDIR /tmp/nmap-7.95
+
+RUN python3 -m pip install build
+RUN apt install python3-venv -y
+RUN apt-get install automake -y
+RUN ./configure && make && sudo make install
+
+WORKDIR /
```

Comment:
Use the builder pattern to not have to ship with the build dependencies.
Use the version as a variable that we can easily change in the future.